SumoLogic.Logging.Common/Sender/SumoLogicMessageSender.cs

```diff
@@ -161,7 +161,11 @@ public async Task Send(string body, string name, string category, string host)
                 }
                 catch (Exception ex)
                 {
-                    if (!(ex is IOException || ex is HttpRequestException || ex is WebException))
+#if NETSTANDARD

```

Comment:
From where is the constant coming from?

Reply:
NETSTANDARD comes from the Microsoft build tools. Just like NET45 exist for Net45 build.